alexandria/core/management/commands/encrypt_files.py

```diff
@@ -36,15 +36,15 @@ def handle(self, *args, **options):
         missing_files = []
 
         # flip between default and encrypted storage to have the correct parameters in the requests
-        DefaultStorage = get_storage_class()
+        Storage = get_storage_class(settings.ALEXANDRIA_FILE_STORAGE)
         for file in tqdm(
             File.objects.filter(
                 Q(encryption_status=File.EncryptionStatus.NOT_ENCRYPTED)
                 | Q(encryption_status__isnull=True)
             ),
         ):
             # get original file content
-            file.content.storage = DefaultStorage()
+            file.content.storage = Storage()
```

Comment:
with renaming `DefaultStorage` to `Storage` the information that it's indeed the default storage is lost but that difference is kind of the point here IMO

Reply:
But it's not the default storage anymore but the storage class that is defined in `ALEXANDRIA_FILE_STORAGE` which is the entire point of this PR